### DISCUSSION TITLE
how add jquery and jquery-ui to nuxt3 plugin?

### DISCUSSION THREAD

**Discussion by MateSoftware2023** [nuxt/nuxt#21009] [Questions]
### Describe the feature

im using nuxt.config way , but work not good.any other way.

 script:[
          {src:"https://code.jquery.com/jquery-3.6.0.js"},
          {src:"https://code.jquery.com/ui/1.13.2/jquery-ui.js"},
      ]

### Additional information

- [X] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by danielroe** [comment]
Would you provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Comment by MateSoftware2023** [comment]
> Would you provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

just want to know how add jquery plugin to nuxt project! not cdn

**Comment by danielroe** [comment]
What are you trying, that isn't working?

**Comment by Gypsy-147** [comment]
i want to know too
